doc/user-guide/Getting-started.md

```diff
@@ -1,140 +1,355 @@
-## Installation
+In this short guide we will set MongooseIM up and get users chatting right away.
+This will guide also you through basic operation, and validation.
+Training and getting to know MongooseIM is the goal here.
```

Comment:
Alt:
In this short guide we will set MongooseIM up and get users chatting right away.
Our goal is to get to know MongooseIM, set it up, go through basic operations and validation.